README.md

```diff
@@ -1,9 +1,13 @@
 # Optimism Integration
+
 A single repository intended to provide the ability to run a local
 Optimistic Ethereum enviornment including both L1 & L2 chains. This
 can be used to rapidly iterate over the many Optimism repositories and
 run integration tests.
 
+A single repository intended to provide the ability to rapidly iterate over
+the many Optimism repositories and run integration tests.
+
```

Comment:
Can remove this because I changed the text over the break & so now this is repeated info.

(changed the text so it's more obvious to people reading the README that this will allow them to spin up an L1 & L2)